### PR TITLE
Update .yaml files for all projects in this repo

### PR DESCRIPTION
**Pull request by isaidamier**

### CODE REVIEW

File: BluetoothChat/.google/packaging.yaml [hunk 1]
```diff
@@ -1,20 +1,47 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 
 # GOOGLE SAMPLE PACKAGING DATA
 #
 # This file is used by Google as part of our samples packaging process.
 # End users may safely ignore this file. It has no relevance to other systems.
 ---
+# Values: {DRAFT | PUBLISHED | INTERNAL | DEPRECATED | SUPERCEDED}
```
Comment (codingjeremy) [inline BluetoothChat/.google/packaging.yaml:21]: remove

File: BluetoothChat/.google/packaging.yaml [hunk 2]
```diff
@@ -1,20 +1,47 @@
...
 status:       PUBLISHED
+
+# Optional, put additional explanation here for DEPRECATED or SUPERCEDED.
+# statusNote:
+
+# See http://go/sample-categories
```
Comment (codingjeremy) [inline BluetoothChat/.google/packaging.yaml:27]: remove

File: BluetoothChat/.google/packaging.yaml [hunk 3]
```diff
@@ -1,20 +1,47 @@
...
 technologies: [Android]
 categories:   [Connectivity]
 languages:    [Java]
 solutions:    [Mobile]
+
+# May be omitted if unpublished
```
Comment (codingjeremy) [inline BluetoothChat/.google/packaging.yaml:33]: remove

File: BluetoothChat/.google/packaging.yaml [hunk 4]
```diff
@@ -1,20 +1,47 @@
...
 github:       android/connectivity
+
+# Values: BEGINNER | INTERMEDIATE | ADVANCED | EXPERT
```
Comment (codingjeremy) [inline BluetoothChat/.google/packaging.yaml:36]: remove

File: BluetoothLeChat/.google/packaging.yaml [hunk 5]
```diff
@@ -1,19 +1,47 @@
...
 # This file is used by Google as part of our samples packaging process.
 # End users may safely ignore this file. It has no relevance to other systems.
 ---
+# Values: {DRAFT | PUBLISHED | INTERNAL | DEPRECATED | SUPERCEDED}
 status:       PUBLISHED
+
+# Optional, put additional explanation here for DEPRECATED or SUPERCEDED.
+# statusNote:
+
+# See http://go/sample-categories
```
Comment (codingjeremy) [inline BluetoothLeChat/.google/packaging.yaml:27]: remove

File: BluetoothLeChat/.google/packaging.yaml [hunk 6]
```diff
@@ -1,19 +1,47 @@
...
 technologies: [Android]
 categories:   [Connectivity]
 languages:    [Java]
 solutions:    [Mobile]
+
+# May be omitted if unpublished
```
Comment (codingjeremy) [inline BluetoothLeChat/.google/packaging.yaml:33]: remove

File: BluetoothLeChat/.google/packaging.yaml [hunk 7]
```diff
@@ -1,19 +1,47 @@
...
 github:       android/connectivity
+
+# Values: BEGINNER | INTERMEDIATE | ADVANCED | EXPERT
```
Comment (codingjeremy) [inline BluetoothLeChat/.google/packaging.yaml:36]: remove

File: BluetoothLeChat/.google/packaging.yaml [hunk 8]
```diff
@@ -1,19 +1,47 @@
...
+
 apiRefs:
-    - android.bluetooth.BluetoothAdapter
+    - ndroid:android.bluetooth.BluetoothAdapter
     - android:android.bluetooth.le.AdvertiseCallback
     - android:android.bluetooth.le.AdvertiseData
     - android:android.bluetooth.le.AdvertiseSettings
     - android:android.bluetooth.le.BluetoothLeAdvertiser
+# Default: apache2. May be omitted for most samples.
+# Alternatives: apache2-android (for AOSP)
```
Comment (codingjeremy) [inline BluetoothLeChat/.google/packaging.yaml:46]: remove

File: BluetoothLeGatt/.google/packaging.yaml [hunk 9]
```diff
@@ -1,18 +1,45 @@
...
 # This file is used by Google as part of our samples packaging process.
 # End users may safely ignore this file. It has no relevance to other systems.
 ---
+# Values: {DRAFT | PUBLISHED | INTERNAL | DEPRECATED | SUPERCEDED}
 status:       PUBLISHED
+
+# Optional, put additional explanation here for DEPRECATED or SUPERCEDED.
+# statusNote:
+
+# See http://go/sample-categories
```
Comment (codingjeremy) [inline BluetoothLeGatt/.google/packaging.yaml:27]: remove

File: BluetoothLeGatt/.google/packaging.yaml [hunk 10]
```diff
@@ -1,18 +1,45 @@
...
 technologies: [Android]
 categories:   [Connectivity]
 languages:    [Java]
 solutions:    [Mobile]
+
+# May be omitted if unpublished
 github:       android/connectivity
-level:        ADVANCED
+
+# Values: BEGINNER | INTERMEDIATE | ADVANCED | EXPERT
```
Comment (codingjeremy) [inline BluetoothLeGatt/.google/packaging.yaml:36]: remove

File: BluetoothLeGatt/.google/packaging.yaml [hunk 11]
```diff
@@ -1,18 +1,45 @@
...
+# Optional, put additional explanation here for DEPRECATED or SUPERCEDED.
+# statusNote:
+
+# See http://go/sample-categories
 technologies: [Android]
 categories:   [Connectivity]
 languages:    [Java]
 solutions:    [Mobile]
+
+# May be omitted if unpublished
```
Comment (codingjeremy) [inline BluetoothLeGatt/.google/packaging.yaml:33]: remove

File: BluetoothLeGatt/.google/packaging.yaml [hunk 12]
```diff
@@ -1,18 +1,45 @@
...
+
+# Values: BEGINNER | INTERMEDIATE | ADVANCED | EXPERT
+level:        INTERMEDIATE
+
 icon:         screenshots/icon-web.png
 apiRefs:
     - android:android.app.Service
     - android:android.bluetooth.BluetoothGattService
+# Default: apache2. May be omitted for most samples.
+# Alternatives: apache2-android (for AOSP)
```
Comment (codingjeremy) [inline BluetoothLeGatt/.google/packaging.yaml:44]: remove

File: CronetKotlin/.google/packaging.yaml [hunk 13]
```diff
@@ -1,18 +1,46 @@
...
 # This file is used by Google as part of our samples packaging process.
 # End users may safely ignore this file. It has no relevance to other systems.
 ---
+# Values: {DRAFT | PUBLISHED | INTERNAL | DEPRECATED | SUPERCEDED}
 status:       PUBLISHED
+
+# Optional, put additional explanation here for DEPRECATED or SUPERCEDED.
+# statusNote:
+
+# See http://go/sample-categories
```
Comment (codingjeremy) [inline CronetKotlin/.google/packaging.yaml:27]: remove

File: CronetKotlin/.google/packaging.yaml [hunk 14]
```diff
@@ -1,18 +1,46 @@
...
 technologies: [Android]
 categories:   [Connectivity]
-languages:    [Java]
+languages:    [Kotlin]
 solutions:    [Mobile]
+
+# May be omitted if unpublished
```
Comment (codingjeremy) [inline CronetKotlin/.google/packaging.yaml:33]: remove

File: CronetKotlin/.google/packaging.yaml [hunk 15]
```diff
@@ -1,18 +1,46 @@
...
 github:       android/connectivity
+
+# Values: BEGINNER | INTERMEDIATE | ADVANCED | EXPERT
```
Comment (codingjeremy) [inline CronetKotlin/.google/packaging.yaml:36]: remove

File: CronetKotlin/.google/packaging.yaml [hunk 16]
```diff
@@ -1,18 +1,46 @@
...
 level:        ADVANCED
+
 apiRefs:
     - android:org.chromium.net.CronetEngine
     - android:org.chromium.net.CronetException
     - android:org.chromium.net.UrlRequest
     - android:org.chromium.net.UrlResponseInfo
+# Default: apache2. May be omitted for most samples.
+# Alternatives: apache2-android (for AOSP)
```
Comment (codingjeremy) [inline CronetKotlin/.google/packaging.yaml:45]: remove

File: NearbyConnectionsCpp/.google/packaging.yaml [hunk 17]
```diff
@@ -1,16 +1,43 @@
...
 # This file is used by Google as part of our samples packaging process.
 # End users may safely ignore this file. It has no relevance to other systems.
 ---
+# Values: {DRAFT | PUBLISHED | INTERNAL | DEPRECATED | SUPERCEDED}
 status:       PUBLISHED
+
+# Optional, put additional explanation here for DEPRECATED or SUPERCEDED.
+# statusNote:
+
+# See http://go/sample-categories
```
Comment (codingjeremy) [inline NearbyConnectionsCpp/.google/packaging.yaml:27]: remove

File: NearbyConnectionsCpp/.google/packaging.yaml [hunk 18]
```diff
@@ -1,16 +1,43 @@
...
 technologies: [Android]
 categories:   [Connectivity]
 languages:    [Java]
 solutions:    [Mobile]
+
+# May be omitted if unpublished
```
Comment (codingjeremy) [inline NearbyConnectionsCpp/.google/packaging.yaml:33]: remove

File: NearbyConnectionsCpp/.google/packaging.yaml [hunk 19]
```diff
@@ -1,16 +1,43 @@
...
 github:       android/connectivity
+
+# Values: BEGINNER | INTERMEDIATE | ADVANCED | EXPERT
```
Comment (codingjeremy) [inline NearbyConnectionsCpp/.google/packaging.yaml:36]: remove

File: NearbyConnectionsCpp/.google/packaging.yaml [hunk 20]
```diff
@@ -1,16 +1,43 @@
...
 level:        ADVANCED
+
 apiRefs:
     - google:com.google.android.gms.nearby.Nearby
+# Default: apache2. May be omitted for most samples.
+# Alternatives: apache2-android (for AOSP)
```
Comment (codingjeremy) [inline NearbyConnectionsCpp/.google/packaging.yaml:42]: remove

File: NearbyConnectionsRockPaperScissors/.google/packaging.yaml [hunk 21]
```diff
@@ -1,16 +1,41 @@
...
 # This file is used by Google as part of our samples packaging process.
 # End users may safely ignore this file. It has no relevance to other systems.
 ---
+# Values: {DRAFT | PUBLISHED | INTERNAL | DEPRECATED | SUPERCEDED}
 status:       PUBLISHED
+
+# Optional, put additional explanation here for DEPRECATED or SUPERCEDED.
+# statusNote:
+
+# See http://go/sample-categories
```
Comment (codingjeremy) [inline NearbyConnectionsRockPaperScissors/.google/packaging.yaml:27]: remove

File: NearbyConnectionsRockPaperScissors/.google/packaging.yaml [hunk 22]
```diff
@@ -1,16 +1,41 @@
...
 technologies: [Android]
 categories:   [Connectivity]
 languages:    [Java]
 solutions:    [Mobile]
+
+# May be omitted if unpublished
```
Comment (codingjeremy) [inline NearbyConnectionsRockPaperScissors/.google/packaging.yaml:33]: remove

File: NearbyConnectionsRockPaperScissors/.google/packaging.yaml [hunk 23]
```diff
@@ -1,16 +1,41 @@
...
 github:       android/connectivity
+
+# Values: BEGINNER | INTERMEDIATE | ADVANCED | EXPERT
```
Comment (codingjeremy) [inline NearbyConnectionsRockPaperScissors/.google/packaging.yaml:36]: remove

File: NearbyConnectionsRockPaperScissors/.google/packaging.yaml [hunk 24]
```diff
@@ -26,4 +51,6 @@ apiRefs:
     - google:com.google.android.gms.nearby.connection.PayloadTransferUpdate
     - google:com.google.android.gms.nearby.connection.PayloadTransferUpdate.Status
     - google:com.google.android.gms.nearby.connection.Strategy
+# Default: apache2. May be omitted for most samples.
+# Alternatives: apache2-android (for AOSP)
```
Comment (codingjeremy) [inline NearbyConnectionsRockPaperScissors/.google/packaging.yaml:55]: remove

File: NearbyConnectionsWalkieTalkie/.google/packaging.yaml [hunk 25]
```diff
@@ -1,16 +1,41 @@
...
 # This file is used by Google as part of our samples packaging process.
 # End users may safely ignore this file. It has no relevance to other systems.
 ---
+# Values: {DRAFT | PUBLISHED | INTERNAL | DEPRECATED | SUPERCEDED}
 status:       PUBLISHED
+
+# Optional, put additional explanation here for DEPRECATED or SUPERCEDED.
+# statusNote:
+
+# See http://go/sample-categories
```
Comment (codingjeremy) [inline NearbyConnectionsWalkieTalkie/.google/packaging.yaml:27]: remove

File: NearbyConnectionsWalkieTalkie/.google/packaging.yaml [hunk 26]
```diff
@@ -1,16 +1,41 @@
...
 technologies: [Android]
 categories:   [Connectivity]
 languages:    [Java]
 solutions:    [Mobile]
+
+# May be omitted if unpublished
```
Comment (codingjeremy) [inline NearbyConnectionsWalkieTalkie/.google/packaging.yaml:33]: remove

File: NearbyConnectionsWalkieTalkie/.google/packaging.yaml [hunk 27]
```diff
@@ -26,4 +51,6 @@ apiRefs:
     - google:com.google.android.gms.nearby.connection.PayloadTransferUpdate
     - google:com.google.android.gms.nearby.connection.PayloadTransferUpdate.Status
     - google:com.google.android.gms.nearby.connection.Strategy
+# Default: apache2. May be omitted for most samples.
+# Alternatives: apache2-android (for AOSP)
```
Comment (codingjeremy) [inline NearbyConnectionsWalkieTalkie/.google/packaging.yaml:55]: remove

File: NearbyMessagesBackgroundBeacons/.google/packaging.yaml [hunk 28]
```diff
@@ -1,16 +1,41 @@
...
 # This file is used by Google as part of our samples packaging process.
 # End users may safely ignore this file. It has no relevance to other systems.
 ---
+# Values: {DRAFT | PUBLISHED | INTERNAL | DEPRECATED | SUPERCEDED}
 status:       PUBLISHED
+
+# Optional, put additional explanation here for DEPRECATED or SUPERCEDED.
+# statusNote:
+
+# See http://go/sample-categories
```
Comment (codingjeremy) [inline NearbyMessagesBackgroundBeacons/.google/packaging.yaml:27]: remove

File: NearbyMessagesBackgroundBeacons/.google/packaging.yaml [hunk 29]
```diff
@@ -1,16 +1,41 @@
...
 technologies: [Android]
 categories:   [Connectivity]
 languages:    [Java]
 solutions:    [Mobile]
+
+# May be omitted if unpublished
```
Comment (codingjeremy) [inline NearbyMessagesBackgroundBeacons/.google/packaging.yaml:33]: remove

File: NearbyMessagesBackgroundBeacons/.google/packaging.yaml [hunk 30]
```diff
@@ -1,16 +1,41 @@
...
 github:       android/connectivity
+
+# Values: BEGINNER | INTERMEDIATE | ADVANCED | EXPERT
```
Comment (codingjeremy) [inline NearbyMessagesBackgroundBeacons/.google/packaging.yaml:36]: remove

File: NearbyMessagesBackgroundBeacons/.google/packaging.yaml [hunk 31]
```diff
@@ -1,16 +1,41 @@
...
 level:        ADVANCED
+
```
Comment (codingjeremy) [inline NearbyMessagesBackgroundBeacons/.google/packaging.yaml:38]: remove

File: NearbyMessagesBackgroundBeacons/.google/packaging.yaml [hunk 32]
```diff
@@ -20,4 +45,6 @@ apiRefs:
     - google:com.google.android.gms.nearby.messages.NearbyPermissions
     - google:com.google.android.gms.nearby.messages.Strategy
     - google:com.google.android.gms.nearby.messages.SubscribeOptions
+# Default: apache2. May be omitted for most samples.
+# Alternatives: apache2-android (for AOSP)
```
Comment (codingjeremy) [inline NearbyMessagesBackgroundBeacons/.google/packaging.yaml:49]: remove

File: NearbyMessagesDevices/.google/packaging.yaml [hunk 33]
```diff
@@ -1,16 +1,41 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 
 # GOOGLE SAMPLE PACKAGING DATA
 #
 # This file is used by Google as part of our samples packaging process.
 # End users may safely ignore this file. It has no relevance to other systems.
 ---
+# Values: {DRAFT | PUBLISHED | INTERNAL | DEPRECATED | SUPERCEDED}
```
Comment (codingjeremy) [inline NearbyMessagesDevices/.google/packaging.yaml:21]: remove

File: NearbyMessagesDevices/.google/packaging.yaml [hunk 34]
```diff
@@ -1,16 +1,41 @@
...
 status:       PUBLISHED
+
+# Optional, put additional explanation here for DEPRECATED or SUPERCEDED.
+# statusNote:
+
+# See http://go/sample-categories
```
Comment (codingjeremy) [inline NearbyMessagesDevices/.google/packaging.yaml:27]: remove

File: NearbyMessagesDevices/.google/packaging.yaml [hunk 35]
```diff
@@ -1,16 +1,41 @@
...
 technologies: [Android]
 categories:   [Connectivity]
 languages:    [Java]
 solutions:    [Mobile]
+
+# May be omitted if unpublished
```
Comment (codingjeremy) [inline NearbyMessagesDevices/.google/packaging.yaml:33]: remove

File: NearbyMessagesDevices/.google/packaging.yaml [hunk 36]
```diff
@@ -1,16 +1,41 @@
...
 github:       android/connectivity
+
+# Values: BEGINNER | INTERMEDIATE | ADVANCED | EXPERT
```
Comment (codingjeremy) [inline NearbyMessagesDevices/.google/packaging.yaml:36]: remove

File: NearbyMessagesDevices/.google/packaging.yaml [hunk 37]
```diff
@@ -1,16 +1,41 @@
...
 level:        ADVANCED
+
```
Comment (codingjeremy) [inline NearbyMessagesDevices/.google/packaging.yaml:38]: remove

File: NearbyMessagesDevices/.google/packaging.yaml [hunk 38]
```diff
@@ -20,4 +45,6 @@ apiRefs:
     - google:com.google.android.gms.nearby.messages.NearbyPermissions
     - google:com.google.android.gms.nearby.messages.Strategy
     - google:com.google.android.gms.nearby.messages.SubscribeOptions
+# Default: apache2. May be omitted for most samples.
+# Alternatives: apache2-android (for AOSP)
```
Comment (codingjeremy) [inline NearbyMessagesDevices/.google/packaging.yaml:49]: remove

File: NetworkConnect/.google/packaging.yaml [hunk 39]
```diff
@@ -1,19 +1,46 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 
 # GOOGLE SAMPLE PACKAGING DATA
 #
 # This file is used by Google as part of our samples packaging process.
 # End users may safely ignore this file. It has no relevance to other systems.
 ---
+# Values: {DRAFT | PUBLISHED | INTERNAL | DEPRECATED | SUPERCEDED}
```
Comment (codingjeremy) [inline NetworkConnect/.google/packaging.yaml:21]: remove

File: NetworkConnect/.google/packaging.yaml [hunk 40]
```diff
@@ -1,19 +1,46 @@
...
 status:       PUBLISHED
+
+# Optional, put additional explanation here for DEPRECATED or SUPERCEDED.
+# statusNote:
+
+# See http://go/sample-categories
```
Comment (codingjeremy) [inline NetworkConnect/.google/packaging.yaml:27]: remove

File: NetworkConnect/.google/packaging.yaml [hunk 41]
```diff
@@ -1,19 +1,46 @@
...
 technologies: [Android]
 categories:   [Connectivity]
 languages:    [Java]
 solutions:    [Mobile]
+
+# May be omitted if unpublished
```
Comment (codingjeremy) [inline NetworkConnect/.google/packaging.yaml:33]: remove

File: NetworkConnect/.google/packaging.yaml [hunk 42]
```diff
@@ -1,19 +1,46 @@
...
 github:       android/connectivity
+
+# Values: BEGINNER | INTERMEDIATE | ADVANCED | EXPERT
```
Comment (codingjeremy) [inline NetworkConnect/.google/packaging.yaml:36]: remove

File: NetworkConnect/.google/packaging.yaml [hunk 43]
```diff
@@ -1,19 +1,46 @@
...
 level:        INTERMEDIATE
+
```
Comment (codingjeremy) [inline NetworkConnect/.google/packaging.yaml:38]: remove

File: NetworkConnect/.google/packaging.yaml [hunk 44]
```diff
@@ -1,19 +1,46 @@
...
 icon:         screenshots/icon-web.png
 apiRefs:
     - android:android.os.AsyncTask
     - android:android.net.ConnectivityManager
     - android:android.net.NetworkInfo
+# Default: apache2. May be omitted for most samples.
+# Alternatives: apache2-android (for AOSP)
```
Comment (codingjeremy) [inline NetworkConnect/.google/packaging.yaml:45]: remove

File: WifiRttScan/.google/packaging.yaml [hunk 45]
```diff
@@ -1,16 +1,41 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 
 # GOOGLE SAMPLE PACKAGING DATA
 #
 # This file is used by Google as part of our samples packaging process.
 # End users may safely ignore this file. It has no relevance to other systems.
 ---
+# Values: {DRAFT | PUBLISHED | INTERNAL | DEPRECATED | SUPERCEDED}
```
Comment (codingjeremy) [inline WifiRttScan/.google/packaging.yaml:21]: remove

File: WifiRttScan/.google/packaging.yaml [hunk 46]
```diff
@@ -1,16 +1,41 @@
...
 status:       PUBLISHED
+
+# Optional, put additional explanation here for DEPRECATED or SUPERCEDED.
+# statusNote:
+
+# See http://go/sample-categories
```
Comment (codingjeremy) [inline WifiRttScan/.google/packaging.yaml:27]: remove

File: WifiRttScan/.google/packaging.yaml [hunk 47]
```diff
@@ -1,16 +1,41 @@
...
 technologies: [Android]
 categories:   [Connectivity]
 languages:    [Java]
 solutions:    [Mobile]
+
+# May be omitted if unpublished
```
Comment (codingjeremy) [inline WifiRttScan/.google/packaging.yaml:33]: remove

File: WifiRttScan/.google/packaging.yaml [hunk 48]
```diff
@@ -19,4 +44,6 @@ apiRefs:
     - android:android.net.wifi.rtt.RangingResult
     - android:android.net.wifi.rtt.RangingResultCallback
     - android:android.net.wifi.rtt.WifiRttManager
+# Default: apache2. May be omitted for most samples.
+# Alternatives: apache2-android (for AOSP)
```
Comment (codingjeremy) [inline WifiRttScan/.google/packaging.yaml:48]: remove

File: BeamLargeFiles/.google/packaging.yaml [hunk 49]
```diff
@@ -1,13 +1,41 @@
...
+# See http://go/sample-categories
 technologies: [Android]
 categories:   [Connectivity]
 languages:    [Java]
 solutions:    [Mobile]
+
+# May be omitted if unpublished
 github:       android/connectivity
+
+# Values: BEGINNER | INTERMEDIATE | ADVANCED | EXPERT
```
Comment (codingjeremy) [inline BeamLargeFiles/.google/packaging.yaml:36]: remove

File: BluetoothLeChat/.google/packaging.yaml [hunk 50]
```diff
@@ -1,19 +1,47 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 # GOOGLE SAMPLE PACKAGING DATA
 #
 # This file is used by Google as part of our samples packaging process.
 # End users may safely ignore this file. It has no relevance to other systems.
 ---
+# Values: {DRAFT | PUBLISHED | INTERNAL | DEPRECATED | SUPERCEDED}
```
Comment (codingjeremy) [inline BluetoothLeChat/.google/packaging.yaml:21]: remove

File: BluetoothLeGatt/.google/packaging.yaml [hunk 51]
```diff
@@ -1,18 +1,45 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 
 # GOOGLE SAMPLE PACKAGING DATA
 #
 # This file is used by Google as part of our samples packaging process.
 # End users may safely ignore this file. It has no relevance to other systems.
 ---
+# Values: {DRAFT | PUBLISHED | INTERNAL | DEPRECATED | SUPERCEDED}
```
Comment (codingjeremy) [inline BluetoothLeGatt/.google/packaging.yaml:21]: remove

File: CronetKotlin/.google/packaging.yaml [hunk 52]
```diff
@@ -1,18 +1,46 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 # GOOGLE SAMPLE PACKAGING DATA
 #
 # This file is used by Google as part of our samples packaging process.
 # End users may safely ignore this file. It has no relevance to other systems.
 ---
+# Values: {DRAFT | PUBLISHED | INTERNAL | DEPRECATED | SUPERCEDED}
```
Comment (codingjeremy) [inline CronetKotlin/.google/packaging.yaml:21]: remove

File: NearbyConnectionsCpp/.google/packaging.yaml [hunk 53]
```diff
@@ -1,16 +1,43 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 
 # GOOGLE SAMPLE PACKAGING DATA
 #
 # This file is used by Google as part of our samples packaging process.
 # End users may safely ignore this file. It has no relevance to other systems.
 ---
+# Values: {DRAFT | PUBLISHED | INTERNAL | DEPRECATED | SUPERCEDED}
```
Comment (codingjeremy) [inline NearbyConnectionsCpp/.google/packaging.yaml:21]: remove

File: NearbyConnectionsRockPaperScissors/.google/packaging.yaml [hunk 54]
```diff
@@ -1,16 +1,41 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 
 # GOOGLE SAMPLE PACKAGING DATA
 #
 # This file is used by Google as part of our samples packaging process.
 # End users may safely ignore this file. It has no relevance to other systems.
 ---
+# Values: {DRAFT | PUBLISHED | INTERNAL | DEPRECATED | SUPERCEDED}
```
Comment (codingjeremy) [inline NearbyConnectionsRockPaperScissors/.google/packaging.yaml:21]: remove

File: NearbyConnectionsWalkieTalkie/.google/packaging.yaml [hunk 55]
```diff
@@ -1,16 +1,41 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 
 # GOOGLE SAMPLE PACKAGING DATA
 #
 # This file is used by Google as part of our samples packaging process.
 # End users may safely ignore this file. It has no relevance to other systems.
 ---
+# Values: {DRAFT | PUBLISHED | INTERNAL | DEPRECATED | SUPERCEDED}
```
Comment (codingjeremy) [inline NearbyConnectionsWalkieTalkie/.google/packaging.yaml:21]: remove

File: NearbyMessagesBackgroundBeacons/.google/packaging.yaml [hunk 56]
```diff
@@ -1,16 +1,41 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 
 # GOOGLE SAMPLE PACKAGING DATA
 #
 # This file is used by Google as part of our samples packaging process.
 # End users may safely ignore this file. It has no relevance to other systems.
 ---
+# Values: {DRAFT | PUBLISHED | INTERNAL | DEPRECATED | SUPERCEDED}
```
Comment (codingjeremy) [inline NearbyMessagesBackgroundBeacons/.google/packaging.yaml:21]: remove

File: BluetoothLeChat/.google/packaging.yaml [hunk 57]
```diff
@@ -1,19 +1,47 @@
...
+
+# May be omitted if unpublished
 github:       android/connectivity
+
+# Values: BEGINNER | INTERMEDIATE | ADVANCED | EXPERT
 level:        INTERMEDIATE
+
 apiRefs:
-    - android.bluetooth.BluetoothAdapter
+    - ndroid:android.bluetooth.BluetoothAdapter
```
[the remark this answers is not in the document]
Comment (isaidamier) [reply BluetoothLeChat/.google/packaging.yaml:40]: ack

File: BeamLargeFiles/.google/packaging.yaml [hunk 58]
```diff
@@ -1,13 +1,41 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 
 # GOOGLE SAMPLE PACKAGING DATA
 #
 # This file is used by Google as part of our samples packaging process.
 # End users may safely ignore this file. It has no relevance to other systems.
 ---
+# Values: {DRAFT | PUBLISHED | INTERNAL | DEPRECATED | SUPERCEDED}
```
[the remark this answers is not in the document]
Comment (isaidamier) [reply BeamLargeFiles/.google/packaging.yaml:21]: ack

File: BeamLargeFiles/.google/packaging.yaml [hunk 59]
```diff
@@ -1,13 +1,41 @@
...
 status:       PUBLISHED
+
+# Optional, put additional explanation here for DEPRECATED or SUPERCEDED.
+# statusNote:
+
+# See http://go/sample-categories
```
[the remark this answers is not in the document]
Comment (isaidamier) [reply BeamLargeFiles/.google/packaging.yaml:27]: ack

File: BeamLargeFiles/.google/packaging.yaml [hunk 60]
```diff
@@ -1,13 +1,41 @@
...
 technologies: [Android]
 categories:   [Connectivity]
 languages:    [Java]
 solutions:    [Mobile]
+
+# May be omitted if unpublished
```
[the remark this answers is not in the document]
Comment (isaidamier) [reply BeamLargeFiles/.google/packaging.yaml:33]: ack

File: NearbyConnectionsCpp/.google/packaging.yaml (hunk 53 above)
Comment (isaidamier) [reply NearbyConnectionsCpp/.google/packaging.yaml:21]: ack

File: NearbyConnectionsWalkieTalkie/.google/packaging.yaml (hunk 55 above)
Comment (isaidamier) [reply NearbyConnectionsWalkieTalkie/.google/packaging.yaml]: ack

File: NearbyMessagesBackgroundBeacons/.google/packaging.yaml (hunk 56 above)
Comment (isaidamier) [reply NearbyMessagesBackgroundBeacons/.google/packaging.yaml]: ack

File: NearbyConnectionsRockPaperScissors/.google/packaging.yaml (hunk 54 above)
Comment (isaidamier) [reply NearbyConnectionsRockPaperScissors/.google/packaging.yaml]: ack

File: CronetKotlin/.google/packaging.yaml (hunk 52 above)
Comment (isaidamier) [reply CronetKotlin/.google/packaging.yaml]: ack

File: NearbyMessagesDevices/.google/packaging.yaml [hunk 61]
```diff
@@ -18,24 +18,13 @@
...
 solutions:    [Mobile]
-
-# May be omitted if unpublished
 github:       android/connectivity
-
-# Values: BEGINNER | INTERMEDIATE | ADVANCED | EXPERT
 level:        ADVANCED
-
 apiRefs:
     - google:com.google.android.gms.nearby.Nearby
```
Comment (codingjeremy) [inline NearbyMessagesDevices/.google/packaging.yaml]: All gms core API should be listed like this:

    - gms:com.google.android.gms.location.FusedLocationProviderClient

File: NearbyMessagesDevices/.google/packaging.yaml [hunk 62]
```diff
@@ -45,6 +34,4 @@ apiRefs:
     - google:com.google.android.gms.nearby.messages.NearbyPermissions
     - google:com.google.android.gms.nearby.messages.Strategy
     - google:com.google.android.gms.nearby.messages.SubscribeOptions
```
Comment (codingjeremy) [inline NearbyMessagesDevices/.google/packaging.yaml]: - gms:com.google.android.gms...

File: NearbyConnectionsRockPaperScissors/.google/packaging.yaml [hunk 63]
```diff
@@ -51,6 +40,4 @@ apiRefs:
     - google:com.google.android.gms.nearby.connection.PayloadTransferUpdate
     - google:com.google.android.gms.nearby.connection.PayloadTransferUpdate.Status
     - google:com.google.android.gms.nearby.connection.Strategy
```
Comment (codingjeremy) [inline NearbyConnectionsRockPaperScissors/.google/packaging.yaml]: Correct to gms: ->
    - gms:com.google.android.gms.etc.etc.

File: NearbyConnectionsWalkieTalkie/.google/packaging.yaml [hunk 64]
```diff
@@ -51,6 +40,4 @@ apiRefs:
     - google:com.google.android.gms.nearby.connection.PayloadTransferUpdate
     - google:com.google.android.gms.nearby.connection.PayloadTransferUpdate.Status
     - google:com.google.android.gms.nearby.connection.Strategy
```
Comment (codingjeremy) [inline NearbyConnectionsWalkieTalkie/.google/packaging.yaml]: Correct to gms: ->
    - gms:com.google.android.gms.etc.etc.

File: NearbyMessagesBackgroundBeacons/.google/packaging.yaml [hunk 65]
```diff
@@ -45,6 +34,4 @@ apiRefs:
     - google:com.google.android.gms.nearby.messages.NearbyPermissions
     - google:com.google.android.gms.nearby.messages.Strategy
     - google:com.google.android.gms.nearby.messages.SubscribeOptions
```
Comment (codingjeremy) [inline NearbyMessagesBackgroundBeacons/.google/packaging.yaml]: Correct to gms: ->
    - gms:com.google.android.gms.etc.etc.

**Review by codingjeremy — COMMENTED** [review]
Need to use 'gms' for gms libraries.

File: NearbyConnectionsCpp/.google/packaging.yaml (hunk 53 above)
Comment (codingjeremy) [reply NearbyConnectionsCpp/.google/packaging.yaml:21]: Doesn't look like this was removed?

**Review by codingjeremy — COMMENTED** [review]
Looks like there are still some unresolved comments.

**Pull request closed without merging**